### PR TITLE
dark theme B with carousel 

### PR DESCRIPTION
**Pull request by accelerator-collab**

### CODE REVIEW

**Comment by accelerator-collab** [comment]
Screen Shot 2019-09-29 at 11 17 17 PM

File: src/components/Carousel/Carousel.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const RightArrow = (props) => {
 };
 
 const Slide = ({ image }) => {
+	console.log(image, "images")
```
Comment (pauvalenzuela) [inline src/components/Carousel/Carousel.js]: remove console.log

File: src/styles/_carousel.scss [hunk 2]
```diff
@@ -1,8 +1,10 @@
 .slider {
 	position: relative;
 	width: 100%;
-	margin: 0 auto;
-	height: 500px;
+	// margin: 0 auto;
+	margin-top: 101px;
```
Comment (pauvalenzuela) [inline src/styles/_carousel.scss]: Why do we need this specific values in spacing?

File: src/styles/_common.scss [hunk 3]
```diff
@@ -1 +1,17 @@
-$card-background-color: #f5f5f5;
\ No newline at end of file
+$card-background-color: #f5f5f5;
+
+
+.cta--center--position{
+    margin-left: 40%;
```
Comment (pauvalenzuela) [inline src/styles/_common.scss]: We have our own layout css for position the content inside the container. See my PR in the A-light/B-light.

File: src/styles/_common.scss [hunk 4]
```diff
@@ -1 +1,17 @@
...
+    margin-top: 157px;
+    margin-bottom: 127px;
+
+}
+
+.carousel--center--position{
+    top: 846px;
```
Comment (pauvalenzuela) [inline src/styles/_common.scss]: Refrain using absolute position and specific values on it. Used our spacing css. Thank you.

**Pull request closed without merging**